lib/workers/repository/process/extract-update.ts

```diff
@@ -71,7 +72,10 @@ export async function extract(
   const cache = getCache();
   cache.scan ||= {};
   const cachedExtract = cache.scan[baseBranch!];
-  const configHash = hasha(JSON.stringify(config));
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { packageRules, ...remainingConfig } = config;
```

Comment:
I didn't want to clone it unnecessarily (so I could delete the field), couldn't think of a way to do this without the eslint rule being triggered

Reply:
Should we change `ignoreRestSiblings` to true in eslint rules? Currently it is false. 
[docs](https://eslint.org/docs/latest/rules/no-unused-vars#ignorerestsiblings)

Reply:
Should we change ignoreRestSiblings to true in eslint rules? Currently it is false.
[docs](https://eslint.org/docs/latest/rules/no-unused-vars#ignorerestsiblings)

Reply:
Good idea